lib/client.js

```diff
@@ -12,42 +12,35 @@ const telemetryHeader = {
   }
 };
 
-custom.setHttpOptionsDefaults({
-  headers: {
-    'User-Agent': `${pkg.name}/${pkg.version}`,
-    'Auth0-Client': Buffer.from(JSON.stringify(telemetryHeader)).toString('base64')
-  },
-  timeout: 4000
```

Comment:
Moved this below to use configuration options but removed the `timeout` value here.